### PR TITLE
Fix misspellings using 'codespell' tool

### PR DESCRIPTION
**Pull request by anatol**

### CODE REVIEW

**Comment by ChuckM** [comment]
:+1: Looks good to me.

File: lib/cm3/dwt.c [hunk 1]
```diff
@@ -27,7 +27,7 @@
  * benchmarking performance of the code. If function fails, the cycle counter
  * isn't available on this architecture.
  *
- * @returnd true, if success
+ * @returned true, if success
```
Comment (fenugrec) [inline lib/cm3/dwt.c]: Almost - this is a doxyblock so it needs to be "@return"

File: doc/Doxyfile_common [hunk 2]
```diff
@@ -1674,7 +1674,7 @@ UML_LOOK               = NO
 # the class node. If there are many fields or methods and many nodes the
 # graph may become too big to be useful. The UML_LIMIT_NUM_FIELDS
 # threshold limits the number of items for each type to make the size more
-# managable. Set this to 0 for no limit. Note that the threshold may be
+# manageable. Set this to 0 for no limit. Note that the threshold may be
```
Comment (karlp) [inline doc/Doxyfile_common]: I'll probably skip this one, as I think this is a typo in upstream example code.

**Comment by karlp** [comment]
Looks good, will merge "soon"

**Comment by anatol** [comment]
Any update?

**Comment by karlp** [comment]
Thanks for the reminder, merged in ae41782  :)

**Pull request merged**